lib/src/client_messages.dart

```diff
@@ -36,7 +35,7 @@ abstract class ClientMessage {
   }
 }
 
-void _applyStringToBuffer(UTF8BackedString string, ByteDataWriter buffer) {
+void applyStringToBuffer(UTF8BackedString string, ByteDataWriter buffer) {
```

Comment:
Hm, maybe this method should be part of `UTF8BackedString`?

Reply:
I think so :)